Documentation/ceph-upgrade.md

```diff
@@ -529,9 +532,26 @@ are updated.
 
 ```console
 # kubectl --namespace rook-ceph get pod -o jsonpath='{range .items[*]}{range .spec.containers[*]}{.image}{"\n"}' -l 'app in (csi-rbdplugin,csi-rbdplugin-provisioner,csi-cephfsplugin,csi-cephfsplugin-provisioner)' | sort | uniq
-quay.io/cephcsi/cephcsi:v1.2.2
+quay.io/cephcsi/cephcsi:v2.0.1
 quay.io/k8scsi/csi-attacher:v1.2.0
 quay.io/k8scsi/csi-node-driver-registrar:v1.2.0
 quay.io/k8scsi/csi-provisioner:v1.4.0
 quay.io/k8scsi/csi-snapshotter:v1.2.2
+quay.io/k8scsi/csi-resizer:v0.4.0
 ```
+### Updating cephcsi from v2.0.0 to v2.0.1
+
+If you have a `v2.0.0` cluster, nodes with mounted Ceph persistent volumes will [hang after reboot](ceph-common-issues.md#node-hangs-after-reboot).
+This is fixed by updating `cephcsi` to `v2.0.1`.
+
+The steps below detail how to verify the update is successful and
+were tested on a cluster with [Ceph Block Storage](ceph-block.md).
+
+#### Update process
+
+Follow the steps in [custom images](#use-custom-images) to update
+`cephcsi:v2.0.0` to `cephcsi:v2.0.1`.
+
+All nodes with application pods using Ceph persistent volumes should be drained.
```

Comment:
What happens if they don't drain? They will run into the hang if they reboot the node? Won't they be draining the pods anyway when they restart a node? I still wonder if we really need this step in the upgrade.

Reply:
If they don't drain the node, the reboot will hang till time out for the first time.  One the it's drained or rebooted we may not see this issue. ( Need to verify)